pkg/controller/controller_binding.go

```diff
@@ -34,6 +34,9 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+// bindingControllerKind contains the schema.GroupVersionKind for this controller type.
+var bindingControllerKind = v1alpha1.SchemeGroupVersion.WithKind("Binding")
```

Comment:
I just followed the pattern from [deployment_controller.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/deployment/deployment_controller.go#L64), [replication_controller.g](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/replication/replication_controller.go#L60) and so on